assets/scripts/dialogs/AnalyticsDialog.jsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * Street Analytics (dialog box)
+ *
+ * Renders the "Analytics" dialog box.
+ *
+ */
+import React, { useEffect } from 'react'
+import { connect } from 'react-redux'
+import { FormattedMessage } from 'react-intl'
+import PropTypes from 'prop-types'
+import Dialog from './Dialog'
+import SegmentAnalytics from './Analytics/SegmentAnalytics'
+import { numberWithCommas } from '../util/formatting'
+import { trackEvent } from '../app/event_tracking'
+import './AnalyticsDialog.scss'
+
+const NO_CAPACITY = { average: 0, potential: 0 }
+
+const CAPACITIES = {
+  sidewalk: { average: 15000, potential: 15000 },
+  'bus-lane': { average: 5000, potential: 5000 },
+  'drive-lane': { average: 1500, potential: 2000 },
+  'turn-lane': { average: 1500, potential: 2000 },
+  'bike-lane': { average: 12000, potential: 12000 },
+  'scooter': { average: 12000, potential: 12000 },
+  'streetcar': { average: 18000, potential: 20000 },
+  'light-rail': { average: 18000, potential: 20000 }
+}
+
+const getCapacity = (type) => {
+  return CAPACITIES[type] || NO_CAPACITY
+}
+
+const addSegmentData = (segments) => {
+  return segments.map(item => {
+    return {
+      label: `${item.variantString} ${item.type}`,
+      capacity: getCapacity(item.type),
+      segment: item
+    }
+  })
+}
+
+function AnalyticsDialog (props) {
+  useEffect(() => {
+    trackEvent('Interaction', 'Open analytics dialog box', null, null, false)
+  }, [])
+
+  const segmentData = addSegmentData(props.street.segments)
+
+  const sumFunc = (total, num) => {
+    return total + num
+  }
+
+  const averageTotal = segmentData.map(item => item.capacity.average).reduce(sumFunc)
+  const potentialTotal = segmentData.map(item => item.capacity.potential).reduce(sumFunc)
+
+  return (
+    <Dialog>
+      {(closeDialog) => (
+        <div className="analytics-dialog">
+          <header>
+            <div className="streetmix-logo" />
```

Comment:
Let's remove the logo from this dialog box.

Reply:
No other comments on layout / style for now! It's great to have anything at this point but we'll definitely need to do a full design pass over this.